yscope-compose/demo-assets/init.sh

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+SCRIPT_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+function generate_sample_clp_config {
+    local ip=$(hostname -i)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)